pkg/accountmanager/manager_test.go

```diff
@@ -0,0 +1,193 @@
+package accountmanager
+
+import (
+	"context"
+	"sync"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"antrea.io/nephe/apis/crd/v1alpha1"
+	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
+	"antrea.io/nephe/pkg/cloudprovider"
+	"antrea.io/nephe/pkg/inventory"
+	"antrea.io/nephe/pkg/util"
+)
+
+var _ = Describe("Account Manager", func() {
+	Context("Account manager workflow", func() {
+		var (
+			credentials               = "credentials"
+			testAccountNamespacedName = types.NamespacedName{Namespace: "namespace01", Name: "account01"}
+			testSecretNamespacedName  = types.NamespacedName{Namespace: "namespace01", Name: "secret01"}
+			testCesNamespacedName     = types.NamespacedName{Namespace: "namespace01", Name: "Ces01"}
+			account                   *v1alpha1.CloudProviderAccount
```

Comment:
Reorg declarations this way.

			fakeClient                client.WithWatch
			secret                    *corev1.Secret
			account                   *v1alpha1.CloudProviderAccount
			selector                       *v1alpha1.CloudEntitySelector
			accountManager            *AccountManager
			accountCloudType          runtimev1alpha1.CloudProvider
			pollIntv                  uint

Reply:
Solved.